lib/rspec/matchers/english_phrases.rb

```diff
@@ -0,0 +1,58 @@
+module RSpec
+  module Matchers
+    # Facilitates converting ruby objects to English phrases.
+    module EnglishPhrases
+      class << self
```

Comment:
I have a preference for using `def self.method_name` over opening the singleton class, and that's what we've favored elsewhere. Mind switching it?